.circleci/config.yml

```diff
@@ -95,7 +115,12 @@ jobs:
           name: Testing
           command: make trace_test
           no_output_timeout: 1800s
-              
+      - run:
+          name: Calculating code coverage
+          command: |
+            cargo tarpaulin --no-count --skip-clean --out Xml
+            bash <(curl -s https://codecov.io/bash)
```

Comment:
seem we must pay for codecov.io, bad news 😭

Reply:
I can try coveralls. I haven't tried tarpaulin + coveralls combination before.

Reply:
we can support codecov.io now, if it works well, we will pay for it because it is cheap :-). 

so please go on.

Reply:
@iamxy 

Seem we can push this to our jenkins?

Reply:
I tried coveralls. The coverage works fine, but since we are not using travis-ci, coveralls can not fetch git information (commit/branch) automatically. https://coveralls.io/builds/17226505
In this aspect, codecov is better. https://codecov.io/gh/codeworm96/tikv

I am sorry that I neglected the cost when discussing the issue. :(